packages/test-utils/src/utils.ts

```diff
@@ -237,8 +237,8 @@ export function fastWallet(): Wallet {
     return new Wallet(fastPrivateKey())
 }
 
-export function randomEthereumAddress(): string {
-    return '0x' + crypto.randomBytes(20).toString('hex')
+export function randomEthereumAddress(): EthereumAddress {
+    return toEthereumAddress('0x' + crypto.randomBytes(20).toString('hex'))
```

Comment:
This could be a utility in `streamr/utils`?

Reply:
I would keep it in test-utils. Hard to imagine a use case of the top off my head where generating random Ethereum public addresses would be needed in non-test code?

Reply:
True. We could then just remove `createEthereumAddress` from `client/test/test-utils/utils.ts` and use this `randomEthereumAddress` method in `StorageNodeRegistry2.test.ts`

Reply:
Fixed 4fcf53f6e62e156ed35e9d23d7e26ec71de21465